pkg/agent/controller/networkpolicy/reconciler.go

```diff
@@ -368,6 +369,14 @@ func (r *reconciler) installOFRule(ofRule *types.PolicyRule, npName, npNamespace
 		r.idAllocator.release(ofID)
 		return 0, fmt.Errorf("error installing ofRule %v: %v", ofID, err)
 	}
+
+	// Count up antrea_agent_ingress_networkpolicy_rule_count or antrea_agent_egress_networkpolicy_rule_count
```

Comment:
Looking at it more closely, installOFRule is also invoked for update of network policy rules (through "update" function). We may double count the rules. Please look at "add" function to get the correct count.

Reply:
Thank you, Actually I wanted to confirm the point whether we have a case to increment or decrement counters in update.